api/handler/v1beta1/appeal.go

```diff
@@ -54,7 +54,7 @@ func (s *GRPCServer) ListUserAppeals(ctx context.Context, req *guardianv1beta1.L
 func (s *GRPCServer) ListAppeals(ctx context.Context, req *guardianv1beta1.ListAppealsRequest) (*guardianv1beta1.ListAppealsResponse, error) {
 	filters := &domain.ListAppealsFilter{}
 	if req.GetAccountId() != "" {
-		filters.AccountID = req.GetAccountId()
+		filters.AccountIDs = []string{req.GetAccountId()}
 	}
 	if req.GetStatuses() != nil {
```

Comment:
This also no need to check nil. GetStatuses I think already handle that.

Reply:
done